addons/common/functions/fnc_getVehicleCargo.sqf

```diff
@@ -17,7 +17,7 @@
 
 params [["_vehicle", objNull, [objNull]]];
 
-private _config = configFile >> "CfgVehicles" >> _vehicle;
+private _config = configOf _vehicle;
```

Comment:
How it could even work before the change? 🤔 config lookup should fail is object is passed.

Reply:
I don't even work here, chief.